src/utils/request.js

```diff
@@ -213,3 +213,14 @@ export const me = async (data) => {
 
   return json.error ? false : formatMe(json);
 };
+
+export const editCandidate = async (data, id) => {
+  const res = await fetch(`https://match-labs-api.herokuapp.com/api/candidates/` + id, {
+    method: "PUT",
+    headers: { ...config.authorization, ...config.headers },
+    body: JSON.stringify({ candidate: data }),
```

Comment:
Maybe its my fault because I didn't get to explain the homework properly, but in here it would be nice if we could edit the company as well. We would have to rename the CandidateForm, to AccountForm or something similar, but it would be nice to reuse the component. 

So on this line you would pass the dynamic value of the role.

Reply:
And of course we would have to do a conditional, to edit candidate || company